resolver.go

```diff
@@ -103,7 +104,14 @@ func ResolveFile(fset *gotok.FileSet, filename string, src interface{}) ([]Entry
 
 // ResolveAuthors extracts the authors from the parsed text of a bibtex field,
 // usually the author or editor field.
-func ResolveAuthors(txt *ast.ParsedText) ([]Author, error) {
+func ResolveAuthors(txt *ast.ParsedText) (a []Author, err error) {
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
I'd prefer to avoid handling panics and bubble up the error instead.

Reply:
The problem is that the panic comes from inside of resolveAuthor, because it has no way to return errors. I suppose what you mean is, "let's change the signature of resolveAuthor to return errors", so I'll do that.